src/main.ts

```diff
@@ -2,13 +2,29 @@ import 'source-map-support/register';
 import { logger } from './modules/common/Logger';
 import Stoa from './Stoa';
 
+import { ArgumentParser } from "argparse";
 import express from 'express';
+import shell from 'shelljs';
 import { URL } from 'url';
 
-const port: number = Number(process.env.PORT) || 3836;
-const agora_address: URL = new URL(process.env.AGORA_ENDPOINT || "http://127.0.0.1:2826");
-// TODO: Replace hardcoded "127.0.0.1" with  agora_address.hostname (#106)
-const stoa: express.Application = new Stoa("database", "127.0.0.1", agora_address.port).stoa;
+// Compile JSON Schema to TypeScript Type
+shell.exec("npm run json2ts");
```

Comment:
I don't know if we do typescript conversion here.
I think we can run it as a RUN on the Dockerfile.

Reply:
That's a great idea.
We can run npm run json2ts when we create a docker image.
Thank you.

Reply:
So I tested it after I applied it.
I have confirmed that it works well in the docker environment.
But it could be a problem when we run it on a local computer, not a docker environment.
So I think the current method would be good, what do you think?

Reply:
I found a way to use `npm command prestart` instead of `shell.exec`. I solved it with Henry at IRL.
And applied it.